cccatalog-api/cccatalog/api/serializers/image_serializers.py

```diff
@@ -208,12 +208,10 @@ def validate_page_size(value):
     @staticmethod
     def validate_source(input_sources):
         allowed_sources = list(get_sources('image').keys())
-
-        for input_source in input_sources.split(','):
-            if input_source not in allowed_sources:
-                raise serializers.ValidationError(
-                    f"Source \'{input_source}\' does not exist."
-                )
+        input_sources = input_sources.split(',')
+        input_sources= [input_source for input_source in input_sources if input_source in allowed_sources] 
```

Comment:
It looks like this fails the style check because the line is too long. Could you fix that?

The main tests are failing because of an unrelated bug I introduced; if you fetch the latest changes from `master` they should be fixed.